### ISSUE TITLE
date on latex front page use different font

### ISSUE THREAD

**Issue by andreacassioli** [sphinx-doc/sphinx#2540]
I have noticed that the project year displayed on the front page of the latex doc uses a different font than the one used for the title

![screenshot from 2016-05-13 11 19 07](https://cloud.githubusercontent.com/assets/6704112/15243549/8fba3740-18fc-11e6-8234-ea8560dac46d.png)

In my `conf.py` 

```
latex_elements = {
# The paper size ('letterpaper' or 'a4paper').                                                                                                                                                                             
'papersize': 'a4paper',

# The font size ('10pt', '11pt' or '12pt').                                                                                                                                                                                
'pointsize': '9pt',

'fontpkg':'',
# Additional stuff for the LaTeX preamble.                                                                                                                                                                                 
'preamble': ADDITIONAL_PREAMBLE,
}
```

I set `fontpkg` to `''` so to use Computer Modern fonts. My guess is that setting does not applies to the front page.  Any clue??

I run on sphinx 1.4.1

**Comment by jfbu** [comment]
The date uses the document "roman" font, whereas the other things use the bold sans-serif document font (aka `\py@HeaderFamily`). As a work-around to get this to be used for the date too :

``` python
latex_elements = {
# The paper size ('letterpaper' or 'a4paper').                                                                                                                                                                             
'papersize': 'a4paper',

# The font size ('10pt', '11pt' or '12pt').                                                                                                                                                                                
'pointsize': '9pt',

'fontpkg':'',

'maketitle': '\\makeatletter\\py@HeaderFamily\\makeatother\\maketitle\\normalfont',
}
```

or, simpler, `'\\sffamily\\bfseries\\maketitle\\normalfont'` which does the same thing if you don't customize from the preamble the `\py@HeaderFamily` macro.

**Comment by andreacassioli** [comment]
Thank you it works!
I didn't realised I could use that option.

However I find a bit misleading that changing the font affect the title but not the date! Isn't it a bug somehow?

**Comment by jfbu** [comment]
This is how `\maketitle` is defined in `sphinxmanual.cls` and `sphinxhowto.cls`. Arguably, the date should also have been typeset using `\py@HeaderFamily`.

Perhaps it is a bug, but on the bright side this feature allows us to customize separately the font for date, although it is in the non obvious way I indicated.

**Comment by andreacassioli** [comment]
Sure,
I understand.

My point is that either it is clear from the docs that  fontpkg does not affect the date, or the date must have the same font as the title. 

Customizing is always good, but the default behavior in this case seems counter intuitive to me.

**Comment by tk0miya** [comment]
I agree this is a bug.
@jfbu Could you fix this when you have a time?

**Comment by jfbu** [comment]
sorry for having missed 1.4.2 release. The PR #2601 changes the font for the date to use the same as for title etc...  Also for author address, which  `sphinx.sty` allows to define in LaTeX via use of `\authoraddress` command, and the command can switch to another font if wanted.

**Comment by tk0miya** [comment]
Don't mind, it was scheduled release. We would release minor fixes at 4-6 weeks intervals.
But 1.4.3 will be released soon (maybe within a few days), because 1.4.2 has packaging problem...

Anyway, #2601 looks good. Let's merge into next release :-)

**Comment by jfbu** [comment]
I have merged #2601. Brief look at the source of `inputenc` at

    /usr/local/texlive/2016/texmf-dist/source/latex/base/inputenc.dtx

shows this 

    \changes{v1.2a}{2014/04/20}{Added alternative definition for xe/lua tex (pr/4024)}%

which was thus first introduced I guess into TeXLive 2014. And indeed, testing with TeXLive 2013 the following latex document:

```latex
\documentclass{article}
\usepackage[utf8]{inputenc}
\begin{document}
éùà
\end{document}
```
compilation fails.

More recent versions of inputenc (as since TL2014) only print a warning.

**Comment by jfbu** [comment]
I have merged #2601.

**Comment by tk0miya** [comment]
This has been resolved. I close now.
